Jenkinsfile

```diff
@@ -509,3 +538,38 @@ def archiveServerLogs(String mongoDbRealmContainerId, String commandServerContai
 def runCommand(String command){
   return sh(script: command, returnStdout: true).trim()
 }
+
+def build_jvm_linux() {
+    unstash 'packages'
+    docker.build('jvm_linux', '-f packages/cinterop/src/jvmMain/linux/generic.Dockerfile .').inside {
+        sh """
+           cd packages/cinterop/src/jvmMain/linux/
+           rm -rf build-dir
```

Comment:
We could consider allowing incremental builds for non-release builds. But guess it doesn't matter if we only build this for releases.